### DISCUSSION TITLE
My account was suspended, I submitted an  ticket but the person who responded didn't look like it was official

### DISCUSSION THREAD

**Discussion by PengLaiSiShaoXiao** [community/community#131619] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account was suspended at 7.2 and I immediately requested a reinstatement ticket.
But until today the main email (1206670314@qq.com) of my github account has not been restored in any way. I looked through my school mailbox and found that someone had replied to my email.
![Screenshot 2024-07-10 130330](https://github.com/community/community/assets/175183589/181d3188-b4ea-4990-a719-2d08e66c7b45)
But after I replied, I found out that the ticket he listed was not the same as the ticket I submitted the application for, and the email address was very strange (forgive me for never seeing such a long email address).
His ticket is #3655465 and I applied for #2867118.This is strange because I didn't get any reply from GitHub on my main mailbox, but instead I was contacted by an unknown person in my school mailbox.

**Comment by TheGiraffe3** [comment]
Don't trust that email. Try contact the GitHub support instead.
I believe it is support@github.com.
Also see support.github.com, though I'm not sure what it does.
